addons/mail/static/src/core/web/chatter.js

```diff
@@ -231,7 +231,7 @@ export class Chatter extends Component {
     get toRecipientsText() {
         const recipients = [...this.state.thread.recipients].slice(0, 5).map(({ partner }) => {
             const text = partner.email ? partner.emailWithoutDomain : partner.name;
-            return `<span class="text-muted" title="${escapeHTML(partner.email)}">${escapeHTML(
+            return `<span class="text-muted" title="${escape(partner.email)}">${escape(
```

Comment:
Replacing `escapeHTML` by `escape` here, while keeping current implementation of `escape`, is a security issue. Escaping quotes is very important